subsys/bluetooth/host/conn.c

```diff
@@ -1205,28 +1207,40 @@ void bt_conn_set_state(struct bt_conn *conn, bt_conn_state_t state)
 	}
 }
 
-struct bt_conn *bt_conn_lookup_handle(uint16_t handle)
+struct bt_conn *bt_conn_lookup_handle(uint16_t handle, int type)
```

Comment:
`type` should be an `uint8_t`.

Reply:
> `type` should be an `uint8_t`.

If we're going to fine-tune this part, I think it'd be cleaner to just give a name to the enum and use that instead of generic int types.

Reply:
I pushed my suggested solution into a separate commit so that it can be reviewed easier.